src/stepfunctions/steps/service.py

```diff
@@ -0,0 +1,149 @@
+            parameters (dict, optional): The value of this field becomes the effective input for the state.
+            result_path (str, optional): Path specifying the raw input’s combination with or replacement by the state’s result. (default: '$')
+            output_path (str, optional): Path applied to the state’s output after the application of `result_path`, producing the effective output which serves as the raw input for the next state. (default: '$')
+        """
+        kwargs[Field.Resource.value] = 'arn:aws:states:::dynamodb:updateItem'
+        super(DynamoDBUpdateItemStep, self).__init__(state_id, **kwargs)
+
+
+class SnsPublishStep(Task):
+
+    """
+    Creates a Task state to publish a message to SNS topic. See `Call Amazon SNS with Step Functions <https://docs.aws.amazon.com/step-functions/latest/dg/connect-sns.html>`_ for more details.
+    """
+
+    def __init__(self, state_id, wait_for_callback=False, **kwargs):
+        """
+        Args:
+            state_id (str): State name whose length **must be** less than or equal to 128 unicode characters. State names **must be** unique within the scope of the whole state machine.
+            wait_for_callback(bool, optional): Boolean value set to `True` if the Task state should wait for callback to resume the operation. (default: False)
+            timeout_seconds (int, optional): Positive integer specifying timeout for the state in seconds. If the state runs longer than the specified timeout, then the interpreter fails the state with a `States.Timeout` Error Name. (default: 60)
+            heartbeat_seconds (int, optional): Positive integer specifying heartbeat timeout for the state in seconds. This value should be lower than the one specified for `timeout_seconds`. If more time than the specified heartbeat elapses between heartbeats from the task, then the interpreter fails the state with a `States.Timeout` Error Name.
+            comment (str, optional): Human-readable comment or description. (default: None)
+            input_path (str, optional): Path applied to the state’s raw input to select some or all of it; that selection is used by the state. (default: '$')
+            parameters (dict, optional): The value of this field becomes the effective input for the state.
+            result_path (str, optional): Path specifying the raw input’s combination with or replacement by the state’s result. (default: '$')
+            output_path (str, optional): Path applied to the state’s output after the application of `result_path`, producing the effective output which serves as the raw input for the next state. (default: '$')
+        """
+        if wait_for_callback:
+            kwargs[Field.Resource.value] = 'arn:aws:states:::sns:publish.waitForTaskToken'
+        else:
+            kwargs[Field.Resource.value] = 'arn:aws:states:::sns:publish'
+        
+        super(SnsPublishStep, self).__init__(state_id, **kwargs)
+
+
+class SqsSendMessage(Task):
```

Comment:
Do you think it makes sense to rename this class to `SqsSendMessageStep` with the `Step` suffix for consistency? That seems to be the pattern being followed here.

Reply:
Good catch, I've changed it in the next revision.